.github/workflows/tos.yaml

```diff
@@ -6,15 +6,15 @@ on:
       - opened
       - reopened
       - edited
-  issue_comment:
-    types:
-      - created
-      - edited
```

Comment:
When a comment without a command is added, we don't want to cancel a running workflow. To avoid this, I extracted the command logic so that the concurrency only applies when there is an actual validation job running.